tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ListingsSpec.scala

```diff
@@ -256,6 +270,31 @@ final class ListingsSpec extends BaseIntegrationSpec {
       }
     }
 
+    "get the latest revision of previously tagged resources when queried by tag" in {
+      val resource11Self = resourceSelf(ref11, resource11Id)
+      val resource13Id   = s"${config.deltaUri}/resources/$proj13/_/resource13"
+      val resource13Self = resourceSelf(ref13, resource13Id)
+      val expected       = jsonContentOf(
+        "/kg/listings/project/resources-tagged.json",
+        replacements(
+          Bob,
+          "org"          -> org1,
+          "proj1"        -> proj11,
+          "resourceType" -> resourceType,
+          "id1"          -> resource11Id,
+          "self1"        -> resource11Self,
+          "proj2"        -> proj13,
+          "id2"          -> resource13Id,
+          "self2"        -> resource13Self
+        ): _*
+      )
+
+      deltaClient.get[Json](s"/resources?tag=$tag1", Bob) { (json, response) =>
+        response.status shouldEqual StatusCodes.OK
+        filterSearchMetadata(json) should equalIgnoreArrayOrder(expected)
+      }
```

Comment:
Here you are doing a search across all resources, that's why there is a test error. You should do /resources/{org}/{project}: https://bluebrainnexus.io/docs/delta/api/resources-api.html#within-a-project

Reply:
Turns out this intermittently passed locally since I forgot the `eventually` 😓 probably since it's beefier than the GHA worker. Fixed now